target/linux/mediatek/filogic/base-files/etc/board.d/01_leds

```diff
@@ -12,6 +12,12 @@ confiabits,mt7981)
 	ucidef_set_led_netdev "lan3" "lan3" "blue:lan-3" "lan3" "link tx rx"
 	ucidef_set_led_netdev "wan" "wan" "blue:wan" "wan" "link tx rx"
 	;;
+bananapi,bpi-r4)
+	ucidef_set_led_netdev "lan0" "lan0" "mt7530-0:00:green:lan" "lan0" "link tx rx"
```

Comment:
Just curious, can we handle this in the device tree?
```
linux,default-trigger = "mt7530-0:00:link";
```

Reply:
Those triggers (and they become many, too many to be useful) are created by old `CONFIG_LED_TRIGGER_PHY` which should be disabled in favor of `CONFIG_PHYLIB_LEDS` which provides support for LEDs controlled by PHYs as well as offloading the `netdev` trigger.

I've tried setting up the `netdev` trigger by default using device tree (as that what *should* get offloaded to hardware), however, there is a problem:

The default trigger is setup at the time the LED is registered with the system. This happens during probe of the PHY driver, and hence *before* the netdev itself is actually registered. Hence, the trigger will be setup *without* offloading, as the setting up offloading would require the netdev to already be registered.

The simple work around is to set up the trigger later on in userspace...

Reply:
Thanks for your reply. Does the new led trigger support rate matching without mdio (such as sfp)?